views/beta.tt.text

```diff
@@ -19,6 +19,22 @@
 
 .betafeature.updatepage.title=New Create Entries Page
 
+.betafeature.s2foundation.cantadd=Sorry, your account is not eligible to test this feature.
+
+.betafeature.s2foundation.off<<
+<p>Activate several updated components on journal pages, including a new version of the icon browser for paid users and a mobile-friendly cleanup of the site-styled comment pages. The underlying components are already well-tested, but introducing them into a new environment might turn up some bugs.</p>
+
+<p>TODO: post a link for reporting bugs.</p>
```

Comment:
For now, the placeholder can just be "report bugs in dw_beta" and if people make new posts, or comment on an existing one, that's probably fine. I don't imagine it'll be high traffic?